plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryInstanceCleaner.java

```diff
@@ -60,6 +60,35 @@ public void setUp()
         this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
     }
 
+    @Test
+    public void cleanUpDatasets()
+    {
+        LOG.info("Identifying datasets to drop...");
+        // Drop all datasets with our label created before yesterday (in UTC)
+        List<String> selfCreatedDatasets = bigQuerySqlExecutor.getSelfCreatedDatasets();
+        TableResult result = bigQuerySqlExecutor.executeQuery(format("" +
+                        "SELECT schema_name " +
+                        "FROM INFORMATION_SCHEMA.SCHEMATA " +
+                        "WHERE date_diff(current_date(), date(creation_time), DAY) > 1 " +
+                        "AND schema_name IN (%s)",
+                selfCreatedDatasets.stream().collect(joining("','", "'", "'"))));
+        ImmutableList<String> datasetsToDrop = Streams.stream(result.getValues())
```

Comment:
ImmutableList -> List

Reply:
Because it's a test class and we keep things simple in tests? Will do.

Reply:
actually because we never declare variables as `ImmutableXxx`